application/src/main/java/org/opentripplanner/apis/transmodel/mapping/RelativeDirectionMapper.java

```diff
@@ -0,0 +1,33 @@
+package org.opentripplanner.apis.transmodel.mapping;
+
+import org.opentripplanner.model.plan.RelativeDirection;
+
+/**
+ * This mapper makes sure that only those values are returned which have a mapping in the Transmodel API,
+ * as we don't really want to return all of them.
+ */
+public class RelativeDirectionMapper {
+
+  public static RelativeDirection map(RelativeDirection relativeDirection) {
+    return switch (relativeDirection) {
+      case DEPART,
+        SLIGHTLY_LEFT,
+        HARD_LEFT,
+        LEFT,
+        CONTINUE,
+        SLIGHTLY_RIGHT,
+        RIGHT,
+        HARD_RIGHT,
+        CIRCLE_CLOCKWISE,
+        CIRCLE_COUNTERCLOCKWISE,
+        ELEVATOR,
+        UTURN_LEFT,
+        UTURN_RIGHT -> relativeDirection;
+      // for these the Transmodel API doesn't have a mapping. should it?
+      case ENTER_STATION,
+        EXIT_STATION,
+        ENTER_OR_EXIT_STATION,
+        FOLLOW_SIGNS -> RelativeDirection.CONTINUE;
```

Comment:
Sorry for the late response here, but this can be dropped. We will just add the new enum values to the Transmodel GraphQL API - I can do it if you want.

The JS GraphQL lib has a validator to compare to schema files for breaking changes - adding Enum values are considered a dangerous change - not  a braking change. It should be handled by defensive programming on the client side. So, at entur we want to add the new enum values and we will cross fingers and hope our clients did the right thing.

Reply:
I also want to add a better unit-test to the enums we have today - so I can do that in the same change.

Reply:
Could/should we create a CI test for validating that we are not creating breaking schema changes with the JS GraphQL lib?

Reply:
I think we still need the mapper so to we don't ever expose ENTER_OR_EXIT_STATION.